packages/gatsby-source-contentful/src/fetch.js

```diff
@@ -29,9 +53,10 @@ module.exports = async ({ spaceId, accessToken, host, syncToken }) => {
     console.log(
       `Accessing your Contentful space failed. Perhaps you're offline or the spaceId/accessToken is incorrect.`
     )
-    // TODO perhaps continue if there's cached data? That would let
-    // someone develop a contentful site even if not connected to the internet.
-    // For prod builds though always fail if we can't get the latest data.
```

Comment:
I'm not sure what the optimal user workflow is here. I've opted for forcing the user to acknowledge a loss of connectivity rather than trying to auto serve cache if network is down.

Reply:
Opt-in require user to know about this feature in first place and if they don't have connectivity they can't actually look for solution for this online.

There are 2 ways to solve it: 
- do automatic fallback and add console message that cached data is used because plugin couldn't connect with contentful api
- log that plugin can't connect and provide information how to enable this feature

Reply:
My bad, responded to your comment without reading your code change. I think this is better than other solution as this will cause less confusion that possibly stale data is used and require user action means user will acknowledge use of cached data.

Reply:
cool 👍